packages/flame/lib/src/effects/glow_effect.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+
+class GlowEffect extends ComponentEffect<HasPaint> {
+  GlowEffect(this.style, this.blurValue, super.controller, {this.paintId});
+
+  final BlurStyle style;
```

Comment:
Most of the time this would be `outer`, so why not make this parameter optional with the default value of `BlurStyle.outer`?